src/chttpd/src/chttpd.erl

```diff
@@ -362,6 +362,8 @@ catch_error(HttpReq, error, decryption_failed) ->
     send_error(HttpReq, decryption_failed);
 catch_error(HttpReq, error, not_ciphertext) ->
     send_error(HttpReq, not_ciphertext);
+catch_error(HttpReq, error, {erlfdb_error,2101}) ->
+    send_error(HttpReq, {request_transcation_too_large, 2101});
```

Comment:
Nit:
```suggestion
    send_error(HttpReq, {request_transaction_too_large, 2101});
```

Reply:
We could probably just use the `transaction_too_large` atom instead of the tuple `{request_transaction_too_large, 2101}` since the tag explains the error pretty well.

And we'd want to stay somewhat consistent with https://apple.github.io/foundationdb/api-error-codes.html
```
transaction_too_large | 2101 | Transaction exceeds byte limit
```